### PR TITLE
Depreacted: Creating a List Item

### PR DESCRIPTION
**Pull request by aahenry51**
Starting the component.

### CODE REVIEW

**Review by aahenry51 — COMMENTED** [review]
Export of the ListContainer and the style

File: components/ListItem/ListItem.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import {ListContainer} from './ListItem.Container';
+import {styles} from './ListItem.Styles';
+import {FlatList, Text, View} from 'react-native';
+
+const ListItem = () => {
+  return <View style={styles.container}></View>;
+};
+
+export default ListItem;
```
Comment (mshago) [inline components/ListItem/ListItem.js:11]: Do not export by default the component, if you do this you give the chance to renamed the component when is imported.

**Review by mshago — CHANGES_REQUESTED** [review]

File: index.js [hunk 2]
```diff
@@ -1,7 +1,3 @@
-/**
- * @format
- */
-
 import {AppRegistry} from 'react-native';
 import {App} from './App';
 import {name as appName} from './app.json';
```
Comment (raynesmike) [inline index.js:3]: So this is how you rename the import

**Review by raynesmike — APPROVED** [review]

**Comment by Josetf2396** [comment]
looks good will be adding to listviewscreen

**Review by Josetf2396 — APPROVED** [review]

**Comment by aahenry51** [comment]
image

**Comment by aahenry51** [comment]
image

File: src/components/ListItem/ListItem.Style.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+export const style = StyleSheet.create({
+  container: {
+    backgroundColor: '#ffffff',
+    borderColor: '#e5e5e5',
+    borderWidth: 2,
+    borderRadius: 8,
+    height: 70,
+  },
+
```
Comment (raynesmike) [inline src/components/ListItem/ListItem.Style.js:12]: Update this to refere the theme

**Pull request closed without merging**